lib/rails_values/nexl_public_suffix_list.rb

```diff
@@ -0,0 +1,4 @@
+NEXL_IGNORED_PUBLIC_SUFFIX = %w[qld.gov.au sa.gov.au tas.gov.au vic.gov.au wa.gov.au].freeze
+
+NEXL_LIST = PublicSuffix::List.new
+PublicSuffix::List.default.each { |r| NEXL_LIST.add(r) if !NEXL_IGNORED_PUBLIC_SUFFIX.include?(r.value) }
```

Comment:
This will break DataCleaner logic around duplicate suggestions, it uses TLD info to decide if something should be grouped together or not.
Also we should not add Nexl specific changes to this gem as it's a public code. Better we add the exceptions into the code.

Reply:
How is the TLD info used in Data Cleaner. Do you have an example please? 

I think it will have a desirable effect of showing health.qld.gov.au and transport.qld.gov.au as company which can be merged @grantspeelman

Reply:
aaah, I see that governments duplicate finder does not use the TLD info https://github.com/NEXL-LTS/nexl360/blob/main/app/models/data_cleaner/duplicate_domain_finders/public_sector.rb
It is used here though : https://github.com/NEXL-LTS/nexl360/blob/main/app/models/data_cleaner/duplicate_domain_finders/sub_domain.rb

That said, feels wrong to change TLD info as it's not technically right and feels like a "gotcha" waiting to happen. It's also very specific to solving things for Australia and it's RocketReach specific. I think it will be best to have it in the main code base close to the RocketReach code.

Reply:
I will close this PR mostly also because its not a very maintainable solution. There is a little discrepancies though qld.gov.au is treated as a domain but nsw.gov.au is not and its difficult explain it to Francesca let alone Minter.

Reply:
agreed, it's hard to explain 😓